save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/vulnerability/VulnerabilityStatus.kt

```diff
@@ -5,29 +5,33 @@ import kotlinx.serialization.Serializable
 
 /**
  * A value for vulnerability statuses.
+ *
+ * @property value
  */
 @Serializable
 @JsExport
 @Suppress("IDENTIFIER_LENGTH")
-enum class VulnerabilityStatus {
+enum class VulnerabilityStatus(val value: String) {
     /**
      * Approve status
      */
-    APPROVED,
+    APPROVED("Approved"),
 
     /**
      * Automatically approved by batch-upload
      */
-    AUTO_APPROVED,
+    AUTO_APPROVED("Auto-approved"),
 
     /**
      * Review status
      */
-    PENDING_REVIEW,
+    PENDING_REVIEW("Pending review"),
 
     /**
      * Reject status
      */
-    REJECTED,
+    REJECTED("Rejected"),
     ;
+
+    override fun toString(): String = value
```

Comment:
please check that it does not break the database when we save the vulnerability

Reply:
It looks like everything is fine, but I decided to remove it just in case.